cloudomate/util/captchasolver.py

```diff
@@ -0,0 +1,153 @@
+import json, requests
+from time import sleep
+import os
+import base64
+
+"""
+use this already generated API key for testing: fd58e13e22604e820052b44611d61d6c
```

Comment:
Is this tied to an account with real money? If so, I would consider removing it.